### PR TITLE
[DO-2613]Rename "back" to on PEP detailed view page.

### PR DESCRIPTION
**Pull request by rpersten**
…ew page

### CODE REVIEW

File: src/components/pages/datasets/pep/PepDetail.jsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { renderDate, getLocaleField } from 'utils';
 import { ReactRouterPropTypes } from 'utils/prop-types';
 import useTopBarHiddingEffect from 'hooks/useTopBarHiddingEffect';
 import Tooltip from 'components/Tooltip';
+import faqs from 'components/pages/help/faqs';
```
Comment (roman-tiukh) [inline src/components/pages/datasets/pep/PepDetail.jsx:11]: unused import

File: src/components/pages/datasets/pep/PepDetail.jsx [hunk 2]
```diff
@@ -253,7 +254,7 @@ const PepDetail = ({ match, history }) => {
       <div className="flex flex-col block-black">
         <div className="inline-flex mb-3">
           <div className="w-40 lg:w-64 font-bold">{t('pepDetailType')}:</div>
-          <div className="max-w-xl">{pep.pep_type_display}</div>
+          <div className="max-w-xl"><a href="/system/help/">{pep.pep_type_display}</a></div>
```
Comment (roman-tiukh) [inline src/components/pages/datasets/pep/PepDetail.jsx:257]: use react-router Link component

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/components/pages/datasets/pep/PepDetail.jsx (hunk 1 above)
Comment (rpersten) [reply src/components/pages/datasets/pep/PepDetail.jsx]: Done

File: src/components/pages/datasets/pep/PepDetail.jsx (hunk 2 above)
Comment (rpersten) [reply src/components/pages/datasets/pep/PepDetail.jsx]: Done

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**